docs/source/agent-framework/core-service-agents/volttron-central/volttron-central-overview.rst

```diff
@@ -0,0 +1,48 @@
+.. _VOLTTRON-Central:
+
+====================================
+VOLTTRON Central Management Overview
+====================================
+
+VOLTTRON Central is responsible for controlling multiple VOLTTRON instances with a single management instance.
+The mnanaged VOLTTRON instances can be either local or remote. Each managed instance will have a VOLTTRON Central
+Platform agent installed and running to interface with the primary VOLTTRON Central agent.
+
+|VC-VCP Overview|
+
+There is a :ref:`VOLTTRON Central Deployment Demo <VOLTTRON-Central-Deployment>` that will allow you to quickly setup
+and see the current offerings of the interface.
+
+VOLTTRON Central will allow you to:
+
+-  See a list of platforms being managed.
+-  Add and remove platforms.
```

Comment:
Where can you remove a platform?  Is this allow/deny?

Reply:
There is an Web RPC API call that can be made to remove the platform, but it does not seem to be exposed in any convenient way in the VC web interface. Something that could be added, or this could be specified in the doc.